resolver-functions/CreateBulkDownload/CreateBulkDownload.ts

```diff
@@ -124,11 +131,12 @@ export const CreateBulkDownloadResolver = async (root, args, context, info) => {
         }
       }
     `;
-  const res = await fetchFromNextGen({
-    args,
-    context,
-    serviceType: "workflows",
-    customQuery: runBulkDownload,
-  });
-  return res;
+  return (
+    await fetchFromNextGen({
+      args,
+      context,
+      serviceType: "workflows",
+      customQuery: runBulkDownload,
+    })
+  ).data.runWorkflowVersion;
```

Comment:
It is possible that there would be an error from NextGen. Then data and runWorkflowVersion won't exist.

Reply:
Yeah, so as of this PR https://github.com/chanzuckerberg/czid-graphql-federation-server/pull/303 we're now automatically propagating errors from NextGen through Fed's `errors`